# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/wasm/multi-value/regress-1628417.js
# Commit: af2e64962ccf
# Full Hash: af2e64962ccfbdef97762dccbba9dda2ddb03b87
# Author: Andy Wingo <wingo@igalia.com>
# Date: 2020-04-10 02:59:09
# Description:
#   Bug 1628417 - Fix multi-value br stack value shuffling in baseline compiler r=bbouvier
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D70327
# ==============================================================================

diff -r 3ecec71f8dcf -r af2e64962ccf js/src/jit-test/tests/wasm/multi-value/regress-1628417.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/wasm/multi-value/regress-1628417.js	Thu Apr 09 13:43:59 2020 +0000
@@ -0,0 +1,11 @@
+let bytes = wasmTextToBinary(`
+   (module
+     (func $main (export "main") (result i32 i32)
+       (i32.const 1)
+       (i32.const 2)
+       (i32.const 0)
+       (br_table 0 0)))`);
+
+let instance = new WebAssembly.Instance(new WebAssembly.Module(bytes));
+
+instance.exports.main();